prod-sample/sample-modules/late-frank/0.0.0/package-lock.json

```diff
@@ -1328,9 +1328,9 @@
       "dev": true
     },
     "acorn": {
-      "version": "6.4.0",
-      "resolved": "https://registry.npmjs.org/acorn/-/acorn-6.4.0.tgz",
-      "integrity": "sha512-gac8OEcQ2Li1dxIEWGZzsp2BitJxwkwcOm0zHAJLcPJaVvm58FRnk6RkuLRpU1EujipU2ZFODv2P9DLMfnV8mw==",
+      "version": "6.4.1",
```

Comment:
looks like dependabot doesn't also update the minimum version in package.json 😢

Reply:
yep, we might have to do this manually